### PR TITLE
UI edits

### PR DESCRIPTION
**Pull request by Abster101**

### CODE REVIEW

**Comment by Abster101** [comment]
Wanted to show you the integration that I mentioned via Teams but I'm sure there's a better way to go about doing that.

**Comment by Abster101** [comment]
Screen Shot 2021-02-03 at 11 07 21 AM
Screen Shot 2021-02-03 at 11 06 13 AM

File: src/plugins/apresTimeline/plugin.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                creatable: true
+            }
+        );
+        openmct.types.addType(
+            'apres.chronicle.type',
+            {
+                name: 'APRES State Chronicle',
+                cssClass: 'icon-arrow-left',
+                creatable: true,
+
```
Comment (deeptailor) [inline src/plugins/apresTimeline/plugin.js:33]: We need an initialize function like in the activity type, which should have domainObject.configuration, which will include the stateName, startTime and endTime with initial values

**Review by deeptailor — CHANGES_REQUESTED** [review]
I left a comment regarding the issue with the forms and the unknown type

File: src/plugins/apresTimeline/plugin.js (hunk 1 above)
Comment (deeptailor) [reply src/plugins/apresTimeline/plugin.js:33]: With the icon issue, its showing an unknown type in the second picture. Delete these via right click, and create a new chronicle, and see if that works

**Pull request closed without merging**